.github/workflows/TestBinaryInstaller.yml

```diff
@@ -54,3 +54,12 @@ jobs:
         run: |
           ddev exec touch /var/www/html/composer-cache/files/lullabot/drainpipe/bin/task/3.28.0/task_linux_amd64.tar
           ddev composer install
+
+      - name: Test installing without composer cache
+        run: |
+          ddev composer install --no-cache
+          ddev composer install
+          rm -rf vendor
+          ddev composer install
+          rm -rf vendor
+          ddev composer install --no-cache
```

Comment:
I know this tests by making sure there's no errors, but would it make sense to add `test -x vendor/bin/task` too?